github/event_types.go

```diff
@@ -149,6 +149,12 @@ type EditChange struct {
 	Title *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"title,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
+	Note *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
 	Body *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"body,omitempty"`
```

Comment:
@shurcooL inline Changes is very good. I agree for since Changes are not used anywhere else, they keep varying across APIs. I will make these and submit.